src/client/network/network.ts

```diff
@@ -44,4 +50,16 @@ export class Network {
     }
     this.offNUClearMessages.clear()
   }
+
+  /**
+   * Send the given message on the network
+   */
+  send(message: Message, target?: string): void {
```

Comment:
Why make a new Message type with a target as a 2nd parameter here over using NUClearNetSend?

Reply:
Changed to accept a `NUClearNetSend`.